beater/jaeger/http.go

```diff
@@ -42,7 +43,7 @@ const (
 
 var (
 	httpRegistry      = monitoring.Default.NewRegistry("apm-server.jaeger.http")
-	httpMonitoringMap = request.MonitoringMapForRegistry(httpRegistry, monitoringKeys)
+	httpMonitoringMap = request.MonitoringMapForRegistry(httpRegistry, interceptors.MetricsMonitoringKeys)
```

Comment:
given that `interceptors.MetricsMonitoringKeys` is being used in more than just gRPC calls, maybe these keys should be moved somewhere more neutral. maybe `beater/telemetry.go`?